storm-webapp/src/test/java/org/apache/storm/daemon/logviewer/handler/LogviewerLogSearchHandlerTest.java

```diff
@@ -627,6 +627,7 @@ public void testFindNMatches() {
 
     public static class TestDeepSearchLogs {
 
+        public static final int DUMMY_METRICS = 0;
```

Comment:
This name is not great. Why not `openedFiles`?

Reply:
But DUMMY_METRICS doesn't say anything about what it is.

Reply:
Okay.